modules/client-configuration/pages/vhm-azure.adoc

```diff
@@ -35,7 +35,47 @@ This is required for subscription matching to work.
 To see which objects and resources have been inventoried, navigate to menu:Systems[System List > Virtual Systems].
 
 
-Instances running on the Azure public cloud will report this UUID to {productname} Server:
+
+== Assigning Permissions
+
+The VHM you create needs to have the correct permissions assigned, in order for it to access the Azure VM.
+
+Log in to your Azure account as an administrator, and ensure that the Azure user account is in the correct groups.
+You will also need to check that the application itself has the appropriate permissions.
```

Comment:
I'm not sure what you mean by 'roles' here. Are they user roles, and then the permissions are inherited from the role?

Reply:
Basically yes. It is described [here](https://docs.microsoft.com/en-au/azure/role-based-access-control/built-in-roles). But important part is that roles have to be set for application, which is specific in this case. So application is added to the group, so it gets proper role and then it is possible to access VMs via SUMA, because application provides this access.